### PR TITLE
ADD Future Enhancements / Roadmap on the README file

### PR DESCRIPTION
**Pull request by alo7lika**
Closes: #59 

Here's a point-wise summary of the updates made to the README:

- Added Hacktoberfest Banner: A banner was added to promote participation in Hacktoberfest.

- Added GSSoC Banner: A banner was added to highlight participation in the GirlScript Summer of Code.

- Enhanced the README: Improved the overall readability and structure of the README for a better user experience.

- Added Future Enhancements / Roadmap: Introduced a new section outlining upcoming features and improvements planned for Bookstream, including:

1. Wishlist Integration
2. Ratings & Reviews
3. Admin Dashboard
4. Book Recommendations
5. Multilingual Support

Structured the README: I organized the content into clear sections with headings, bullet points, and tables of contents for easier navigation.

This summary captures the significant enhancements made to the README file effectively.

### CODE REVIEW

**Comment by alo7lika** [comment]
@yashksaini-coder I have completed the task. Please review it.
Thank you!

**Comment by yashksaini-coder** [comment]
> @yashksaini-coder I have completed the task. Please review it. Thank you!

Have you read the #24 discussion if not please go through it first. @alo7lika

File: README/329829127-e79eb6de-81b1-4ffb-b6ed-f018bb977e88.png
Comment (yashksaini-coder) [inline README/329829127-e79eb6de-81b1-4ffb-b6ed-f018bb977e88.png:1]: Remove this as this project is not a part of GSSOC-Ext anymore.

File: README.md
Comment (yashksaini-coder) [inline README.md:1]: Can you add more extra badges and social icons like this but more better than these, don't just copy paste this.

image

**Review by yashksaini-coder — CHANGES_REQUESTED** [review]
- Remove the GSSOC banner
- Add more improve image badges & social badges

**Comment by alo7lika** [comment]
> > @yashksaini-coder I have completed the task. Please review it. Thank you!
> 
> Have you read the #24 discussion if not please go through it first. @alo7lika

yes read it

**Comment by yashksaini-coder** [comment]
> > > @yashksaini-coder I have completed the task. Please review it. Thank you!
> > 
> > 
> > Have you read the #24 discussion if not please go through it first. @alo7lika
> 
> yes read it

okh I hope you understand.

**Review by yashksaini-coder — APPROVED** [review]
Its good

**Pull request merged**